js/bittrex.js

```diff
@@ -847,6 +848,23 @@ module.exports = class bittrex extends Exchange {
         }
     }
 
+    parseOrders (orders, market = undefined, since = undefined, limit = undefined, params = {}) {
+        if (this.options['fetchClosedOrdersFilterBySince']) {
+            return super.parseOrders (orders, market, since, limit, params);
+        }
+        const filtered = [];
```

Comment:
Why not just `return super.parseOrders (orders, market, undefined, limit, params);` if the else-case ? )